Makefile

```diff
@@ -3,3 +3,6 @@ lint:
 
 format:
 	@scripts/fmt.sh
```

Comment:
Maybe switch to `golangci-lint run --fix`?

Reply:
I added "make fixlint" for this. Good idea.